MyPlayground.playground/Pages/Array.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,134 @@
+//: [Previous](@previous)
+
+import Foundation
+
+//1
+print("1. First & Last element equal")
+func check_first_last(_ arra: [Int]) -> Bool {
```

Comment:
function name should be in camelcase. do this in all files, 'arra' is not allowed, give proper name.